reports/serializers.py

```diff
@@ -1,8 +1,51 @@
+from typing import List
+
 from reports.models import EnrolmentReport
 from rest_framework import serializers
 
 
+class Size2ArrayFieldBase(serializers.Field):
+    def __init__(self, first_value_name: str = "id", second_value_name: str = "label"):
```

Comment:
Optimally this should allow `kwargs` to support `Field`'s [core arguments](https://www.django-rest-framework.org/api-guide/fields/#core-arguments).